charactersheet/charactersheet/viewmodels/dm/encounter_sections/npc_section.js

```diff
@@ -165,6 +168,15 @@ function NPCSectionViewModel(parentEncounter) {
         self.editFirstModalElementHasFocus(true);
     };
 
+    //Prepopulate methods
+    self.populateRace = function(label, value) {
+        self.blankNPC().race(value);
+    };
+
+    self.populateRaceEdit = function(label, value) {
```

Comment:
Our convention, so far, has been to not add pre-pop to the edit modal. What's the justification for this?